### PR TITLE
allow full log sends

### PR DESCRIPTION
**Pull request by joshblum**
patch does the following:

- adds a hidden option to send 'full logs' (128mb)
- moves the ios logs into a subdir that has background readable permissions so logs can properly rotate
- keep a second log file on mobile when not admin so we can get logs after a rotation

cc @keybase/hotpotatosquad

### CODE REVIEW

File: shared/ios/Keybase/Fs.m [hunk 1]
```diff
@@ -128,12 +128,12 @@ - (NSDictionary*) setupFs:(BOOL)skipLogFile setupSharedHome:(BOOL)setupSharedHom
   // Setup app level directories
   NSString* levelDBPath = [@"~/Library/Application Support/Keybase/keybase.leveldb" stringByExpandingTildeInPath];
   NSString* chatLevelDBPath = [@"~/Library/Application Support/Keybase/keybase.chat.leveldb" stringByExpandingTildeInPath];
-  NSString* logPath = [@"~/Library/Caches/Keybase" stringByExpandingTildeInPath];
+  NSString* logPath = [@"~/Library/Caches/Keybase/logs" stringByExpandingTildeInPath];
```
Comment (chrisnojima) [inline shared/ios/Keybase/Fs.m]: we need some kind of cleanup of old logs story if we're moving the logs around

Comment (joshblum) [reply shared/ios/Keybase/Fs.m]: ah good catch

**Comment by joshblum** [comment]
@chrisnojima this is ready

**Comment by joshblum** [comment]
merging over flake

**Pull request merged**